lib/srv/reexec.go

```diff
@@ -85,6 +85,9 @@ type ExecCommand struct {
 	// policy.
 	UsePAMAuth bool `json:"use_pam_auth"`
 
+	// Environment variables to set for PAM modules.
+	PAMEnvironment map[string]string `json:"pam_environment"`
```

Comment:
nit: it would make sense to move PAM-specific configuration into a value of separate type.

Reply:
adjusted